site/controllers/city.js

```diff
@@ -1,6 +1,12 @@
 const City = require("../models/city");
 
-exports.getUsers = async (req, res, next) => {
+exports.getAddCity = async (req, res, next) => {
```

Comment:
Please fix the name of the method, this method is used to get all the cities, why naming it as `getAddCity`?? Plus since we're not using the `req` and `next` parameters, just delete them

Reply:
The goal of this PR is to fix the university controller.

I just added these functions to get the server started, I will fix them in another PR.